lib/spack/docs/basic_usage.rst

```diff
@@ -1222,11 +1026,11 @@ Module Commands for Shell Scripts
 
 Although Spack is flexible, the ``module`` command is much faster.
 This could become an issue when emitting a series of ``spack load``
-commands inside a shell script.  By adding the ``--shell`` flag,
-``spack module find`` may also be used to generate code that can be
-cut-and-pasted into a shell script.  For example:
+commands inside a shell script.  The ``spack module loads`` may also
+be used to generate code that can be cut-and-pasted into a shell
+script.  For example:
 
-.. code-block:: console
+.. code-block:: sh
```

Comment:
Yeah, we could add a section on .rst and writing documentation. In general, you should use "sh" if you are writing a code-block that you would think of as a shell script. If you are writing a code block that includes a command run on the command line and its subsequent output, you should use console. Otherwise, Sphinx tries to format the output of the command as if it were bash, which isn't what we want.